src/middleware/isValidInvite.js

```diff
@@ -16,7 +16,8 @@ const verify = (req, res, next) => {
 
         
 		if (req.body.email !== user.email) 
-			return next(new AppError('Invalid request email', 403))
+			return next(new AppError(
```

Comment:
the error here was fine, no need to make changes to this file. 

maybe this was done due to my review concerning the test case description in the tests? but yea didn't mean the actual error should change

Reply:
Okay. I have changed it back @jocrah

Reply:
thanks @itzgeebee 

so usually, to preserve the commit history and sometimes save time,  let's prevent manual reverts of files if no changes are no longer needed there.

We could always get the original file from the `main` branch (`git checkout origin/main src/middleware/isValidInvite.js`) and push it. So that it's not subject to any changes we might miss and so on

Reply:
Noted @jocrah

Reply:
Done @jocrah

Reply:
nice, thanks